src/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
The 'class' attribute added to the 'html' tag is not recommended. The 'class' is usually applied to 'body' or any specific html element inside 'body'.